### PR TITLE
Khuzista

### PR DESCRIPTION
**Pull request by KMeem22**
CSS_ECOMMERCE PROJECT.

THIS IS AN AWESOME PROJECT

### CODE REVIEW

**Comment by KMeem22** [comment]
Netlify link: https://candid-starship-bbf818.netlify.app/

File: index.html [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+<body>
+
+    <!----  Header section  --->
+
+    <header class="site_header">
+        <a href="/index.html" class="logo"><img id="logo_img" src="/images/4.jfif" alt="fika"></a>
+
+
+        <nav class="navbar">
```
Comment (slmgnm) [inline index.html:40]: Display the burger menu with media query only for small screens.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+            <div class="fas fa-shopping-cart" id="cart-btn"></div>
+            <div class="fas fa-bars" id="menu-btn"></div>
+        </div>
+
+        <div class="search_form">
+            <input type="search" id="search-box" placeholder="search here ....">
+            <label for="search-box" class="fas fa-search"></label>
+        </div>
+
+        <!--div class="cart_item_container">
```
Comment (slmgnm) [inline index.html]: Remove unnecessary code.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+                    <h5>Korean Coffee Drink</h5>
+                </div>
+
+            </div>
+        </div--->
+
+    </header>
+
+    <br><br><br>
```
Comment (slmgnm) [inline index.html]: Use CSS instead to add margin, padding or gap.

File: menu.html [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+            <input type="search" id="search-box" placeholder="search here ....">
+            <label for="search-box" class="fas fa-search"></label>
+        </div>
+
+    </header>
+
+    <br><br><br>
+
+
+    <section class="product">
```
Comment (slmgnm) [inline menu.html:65]: Display products as flex with CSS, to show them aligned.

File: CSS/style.css [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+
+body{
+    background-color: rgb(103, 46, 34);
+    margin: 8px auto;
+    width: 100%;
+}
+
+html{
+    font-size: 100%;
+    overflow: hidden;
```
Comment (slmgnm) [inline CSS/style.css]: This line is preventing scrolling down, you can remove it : )

File: Khuzista.md [hunk 6]
```diff
@@ -0,0 +1,3 @@
+. I want to make ecommerce page for a coffe shop.
+. I cannot scroll down on the page.
+. i do not undaerstand the problem. I think I did something wrong.
```
Comment (slmgnm) [inline Khuzista.md:3]: -Create a cart page and try to add a product with price and buy button.
-Registration page, login page (2 points)
-Apply any relevant animation in the project (1 point)
-make sure the website is 100% accessibile (2 point)
make sure the website is 100% responsive (2 points)

**Review by slmgnm — COMMENTED** [review]
Please review my comments and let me know if you face problems.

**Pull request closed without merging**